beta/examples/tensorflow/classification/main.py

```diff
@@ -145,6 +146,8 @@ def run(config):
     train_builder, validation_builder = builders
     train_dataset, validation_dataset = datasets
 
+    config = register_default_init_args(config, train_dataset)
```

Comment:
Sample Config is an object that is used in the examples and should not be imported into NNCF. 
```suggestion
    nncf_config = config.nncf_config
    nncf_config = register_default_init_args(nncf_config, train_dataset)
```

Reply:
Fixed